functions/menu.bash

```diff
@@ -19,7 +30,7 @@ show_main_menu() {
   "00 | About openHABian    "    "Information about the openHABian project and this tool" \
   "" "" \
   "01 | Update"                  "Fetch the latest revision of the openHABian Configuration Tool" \
-  "02 | Upgrade System"          "Upgrade all installed software packages to their newest version" \
+  "02 | Upgrade System"          "Upgrade all installed software packages (incl. openHAB2) to their latest version" \
```

Comment:
Probably isn't necessary to state that, as it should be assumed to be included by stating all installed software packages.

Reply:
So I thought, too. But there was criticism, some users didn't understand that. So I think it doesn't do harm to explicitly state that.

Reply:
Ok